### DISCUSSION TITLE
fix the add to studio bug

### DISCUSSION THREAD

**Discussion by ksdavidc** [ScratchAddons/ScratchAddons#4689] [Ideas]
Apparently there is an [add to studio bug](https://scratch.mit.edu/discuss/topic/572279/?page=1#post-5947919). Specifically, on the actual project page of a project, the add to studio button works, but it only displays a limited number (20, to be exact) of studios. (In addition, they are not meaningfully sorted.) This is a real nuisance. 

[On this page](https://scratch.mit.edu/discuss/topic/338174/?page=1#post-6350479/) it is shown that the scratch API allows one to adjust this limit, so it would seem possible to fix this in some way. 

It seems unlikely from the discussion in these posts that the ST is going to address this in the immediate future.

Any possibility of it being fixable via SA?

**Comment by DNin01** [comment]
Scratch does only show 20 studios with no Load more button, but SA could maybe do a little about it, probably with the Full areas addon (it already adds a load more button in the ["What I've been doing" area](https://en.scratch-wiki.info/wiki/File:What_I%27ve_been_doing.png) on profile pages) or just a new addon. If you would like to suggest a possible way to fix the problem, you can [create an issue](https://github.com/ScratchAddons/ScratchAddons/issues/new/choose).

**Reply by WorldLanguages** [reply]
Yeah, I think an issue could be created about this.

**Reply by ksdavidc** [reply]
done. :smile: #4693
